### ISSUE TITLE
Zen mode for mobiles

### ISSUE THREAD

**Issue by AvidSeeker** [excalidraw/excalidraw#5185]
Can zen mode be activated directly through the GUI (e.g: via a clickable button)? the shortcut is helpful for desktop, but not for mobiles where there is no physical keyboard.

**Comment by dwelle** [comment]
You can enable zen mode via context menu, clicking on canvas.

![image](https://user-images.githubusercontent.com/5153846/168046555-c1f30486-51b7-44cc-a059-3ed2aff39c7a.png)

**Comment by AvidSeeker** [comment]
* On mobile devices: managed to enable it by holding down on screen. Nothing changed. [Icons still appear](https://i.imgur.com/odJOwfj.png).
* On tablet devices: holding down didn't bring context menu.

Tried on: Chrome, Samsung Browser, Firefox.

**Comment by dwelle** [comment]
> * On mobile devices: managed to enable it by holding down on screen. Nothing changed. [Icons still appear](https://i.imgur.com/odJOwfj.png).

You're right, I didn't give it much thought. Right now zen mode on mobile doesn't really do anything.

What's your idea of how it should work?

**Comment by AvidSeeker** [comment]
My idea is to add the ability to hide all toolbars and show it on demand by swiping the edge of the screen or clicking on a button.

Many note taking apps (e.g: Samsung notes) implement a similar full screen feature.

That would be super helpful when issues like #5166 are resolved.

**Comment by johnathon-b** [comment]
This feature would also benefit iPad users.

The tool bar should be collapsable and view able with a gesture like swiping from the left side.

On Tablet, the current toolbar is taking up much of the available real estate. When sharing your drawing with another on a computer, it looks like your drawing off centered.

**Comment by amitnovick** [comment]
> * On mobile devices: managed to enable it by holding down on screen. Nothing changed. [Icons still appear](https://i.imgur.com/odJOwfj.png).
> 
>     * On tablet devices: holding down didn't bring context menu on Chrome & Samsung browser. It worked on Firefox.
> 
> 
> Tried on: Chrome, Samsung Browser, Firefox.

I found that you must use the "Cursor" tool in order to be able to bring down the context menu with long press on iPad.
